test/testallfilesdeleted.cpp

```diff
@@ -71,6 +71,7 @@ private slots:
         for (auto it = children.cbegin(); it != children.cend(); ++it) {
             modifier.remove(it.key());
         }
+        QVERIFY(fakeFolder.applyLocalModificationsWithoutSync());
```

Comment:
That was previously done implicit? Or why is it needed now.

Reply:
added a comment